src/styles/index.scss

```diff
@@ -1,3 +1,148 @@
+$colors: (
+  "main-accent": #060b35,
+  "secondary": #616070,
+  "white": #fff,
+  "blue-accent": #00acdc,
+  "gray-element": #f3f3f3,
+  "yellow-stars": #ffde6a
+);
+
+$sizes: (
+  "star-size": 16px,
+  "container-size": 96px,
+);
+
+/* #region card */
+
 body {
   margin: 0;
 }
+
+.card {
+  height: 406px;
+  width: min-content;
+  border: 1px solid map-get($colors, "gray-element");
+  border-radius: 5px;
+  background-color: map-get($colors, "white");
+  font-family: "Roboto", sans-serif;
+  &__rectangle {
```

Comment:
By looking at the selector only I had no idea what it might stand for. Why don't you rename it to something like `__image`?